### PR TITLE
Createuser form

### PR DESCRIPTION
**Pull request by TonnyMusembi**

### CODE REVIEW

File: components/TheCreateUser.vue [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        />
+      </div>
+      <div class="mb-2">
+        <label
+          for=""
+          class="block mb-2 text-sm font-medium text-black-900 dark:text-black-300"
+          >Enter password</label
+        >
+        <input
+          name="new password"
```
Comment (lenimbugua) [inline components/TheCreateUser.vue:44]: use one word like new-password

File: components/TheCreateUser.vue [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          name="phone_number"
+          type="text"
+          class="bg-gray-50 border border-white-300 text-white-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-white-700 dark:border-white-600 dark:placeholder-white-400 dark:text-black dark:focus:ring-blue-500 dark:focus:border-blue-500"
+          placeholder="phone number "
+          required
+        />
+      </div>
+      <div class="mb-2">
+        <label
+          for=""
```
Comment (lenimbugua) [inline components/TheCreateUser.vue:39]: add label for

File: components/TheCreateUser.vue [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          placeholder=" new password"
+          required
+        />
+      </div>
+      <div class="block mb-3">
+        <label for="" class="block text-sm font-medium text-black-900 mb-2"
+          >Confirm password</label
+        >
+        <input
+          name="Confirm new password"
```
Comment (lenimbugua) [inline components/TheCreateUser.vue:56]: change name to confirm-password

File: components/TheCreateUser.vue [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        <input
+          name="new password"
+          type="text"
+          class="bg-gray-50 border border-white-300 text-white-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-white-700 dark:border-white-600 dark:placeholder-white-400 dark:text-black dark:focus:ring-blue-500 dark:focus:border-blue-500"
+          placeholder=" new password"
+          required
+        />
+      </div>
+      <div class="block mb-3">
+        <label for="" class="block text-sm font-medium text-black-900 mb-2"
```
Comment (lenimbugua) [inline components/TheCreateUser.vue:52]: add label for for

**Review by lenimbugua — CHANGES_REQUESTED** [review]

File: components/TheCreateUser.vue (hunk 2 above)
Comment (TonnyMusembi) [reply components/TheCreateUser.vue:39]: ok

File: components/TheCreateUser.vue (hunk 1 above)
Comment (TonnyMusembi) [reply components/TheCreateUser.vue:44]: ok

File: components/TheCreateUser.vue [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        Create User
+      </div>
+      <div class="mb-2">
+        <label
+          for="full_names"
+          class="block mb-2 text-sm font-medium text-black-900 dark:text-black-300"
+          >Full Name</label
+        >
+        <input
+          name="full_names"
```
Comment (lenimbugua) [inline components/TheCreateUser.vue:16]: great!

**Review by lenimbugua — APPROVED** [review]

**Comment by lenimbugua** [comment]
Great!!

**Pull request merged**